sklearn/metrics/classification.py

```diff
@@ -1674,10 +1674,11 @@ def hamming_loss(y_true, y_pred, labels=None, sample_weight=None):
     y_type, y_true, y_pred = _check_targets(y_true, y_pred)
     check_consistent_length(y_true, y_pred, sample_weight)
 
-    if labels is None:
-        labels = unique_labels(y_true, y_pred)
-    else:
-        labels = np.asarray(labels)
+    if labels is not None:
+        warnings.warn(
+            "This parameter was deprecated in version 0.20"
```

Comment:
Actually "was deprecated in 0.21" and can you maybe add "and will be removed in 0.23"?

Reply:
Sure, done..